concrete/controllers/single_page/dashboard/system/files/thumbnails.php

```diff
@@ -12,16 +12,40 @@ public function view()
     {
         $list = Type::getList();
         $this->set('types', $list);
+
     }
 
     public function edit($ftTypeID = false)
     {
         $type = Type::getByID($ftTypeID);
         $this->set('type', $type);
+
+        $this->get_sizing_values();
     }
 
     public function add()
     {
+        $this->get_sizing_values();
+    }
+
+    public function get_sizing_values()
+    {
+        $sizingModes = array(
+            Type::RESIZE_PROPORTIONAL => t('Resize Proportionally'),
+            Type::RESIZE_EXACT => t('Resize and Crop to the Exact Size')
+        );
+        $this->set('sizingModes', $sizingModes);
+
+        $sizingModeHelp = array(
```

Comment:
I did but it didn't change the array.
should I change it manually everywhere I added arrays?